### PR TITLE
Fix deprecation warning

### PR DESCRIPTION
**Pull request by perugini**
Hi, 
I've solved the DEP0106 DeprecationWarning, added crypto.createCipheriv and crypto.createDecipheriv support derived from @vlucas encryption.js, updated modules dependencies and tested with Mocha

### CODE REVIEW

**Comment by perugini** [comment]
Hi @Richienb, any news about it? :)

File: lib/index.js [hunk 1]
```diff
@@ -1,22 +1,31 @@
 'use strict'
 
 module.exports = function (crypto) {
```
Comment (Richienb) [inline lib/index.js:3]: We can modernise the code.

**Comment by Richienb** [comment]
> Hi @Richienb, any news about it? :)

Looks pretty good, just a couple nitpicks.

File: lib/index.js (hunk 1 above)
Comment (perugini) [reply lib/index.js:3]: Good.
I've also updated the README, and added basic.js example file.

File: lib/index.js [hunk 2]
```diff
@@ -88,7 +97,7 @@ module.exports = function (crypto) {
   output.decrypt = function (object, password, config) {
```
Comment (Richienb) [inline lib/index.js:97]: We can use arrow functions and default parameters.

Comment (perugini) [reply lib/index.js]: Done that.

**Comment by perugini** [comment]
Hi @Richienb, can we close it?

**Comment by Richienb** [comment]
Nice work - I'll do some meta tweaks and release this.

**Comment by perugini** [comment]
Good.
I think we can remove the "Looking for a new maintainer", and fix the deprecation warning on npmjs.

**Comment by Richienb** [comment]
Those changes will be made along with the meta tweaks.

**Comment by perugini** [comment]
Ok

**Pull request merged**